skyreader/result.py

```diff
@@ -825,13 +825,41 @@ def create_plot(self, dozoom = False):
 
         print("done.")
 
+    def circular_contour(self, ra, dec, sigma, nside):
```

Comment:
It seems this could be a `@staticmethod` and `self` could be removed as the method does not access any instance/class attribute.

Reply:
Without it it was giving me error